bundle/Resources/config/kernel.yml

```diff
@@ -2,3 +2,9 @@ system:
     default:
         fielddefinition_edit_templates:
             - { template: "EzSystemsRepositoryFormsBundle:ContentType:field_types.html.twig", priority: 0 }
+        location_view:
+            content_name:
+                landing_page:
+                    template: "EzSystemsRepositoryFormsBundle:content_name:content_name.html.twig"
+                    match:
+                        Identifier\ContentType: [landing_page,folder]
```

Comment:
Please try the following:

```yaml
default:
    location_view:
        content_name:
            default_view: 
                template: "EzSystemsRepositoryFormsBundle:content_name:content_name.html.twig"
                match: {}                
```

Furthermore, we should try to use a more specific viewType, e.g. `_platformui_content_name`, to be sure we avoid name collisions.